plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -62,10 +62,16 @@ if ( process.env.CI ) {
 }
 
 export const setupProjects = [
+	{
+		name: 'install wc with wc beta tester',
```

Comment:
Nitpick: I would go with a shorter title, like "install wc". I don't see it relevant to specify it is using the beta tester.

Reply:
The shorter the better 👍 I went with beta tester part because I was thinking about someone who might run the tests (let's say against wp-env) and see the step "install wc" skipped, and might think why WC is skipped and not installed

Reply:
Done - [225b1ac](https://github.com/woocommerce/woocommerce/pull/55063/commits/225b1ac3d70428ca4c52f7a3047b6c273f9098bf)